src/smt/sygus_solver.cpp

```diff
@@ -257,8 +257,11 @@ void SygusSolver::checkSynthSolution(Assertions& as)
 {
   NodeManager* nm = NodeManager::currentNM();
   SkolemManager* sm = nm->getSkolemManager();
-  Notice() << "SygusSolver::checkSynthSolution(): checking synthesis solution"
-           << std::endl;
+  if (isOutputOn(OutputTag::SYGUS))
```

Comment:
This should be a different (new) output tag.  Something like `-o solver`, which summarizes what the (submodules) of SolverEngine are doing.

Reply:
Done.